src/main/java/ch/njol/skript/structures/StructFunction.java

```diff
@@ -58,45 +60,56 @@ public class StructFunction extends Structure {
 
 	public static final Priority PRIORITY = new Priority(400);
 
+	private static final Pattern SIGNATURE_PATTERN =
+			Pattern.compile("(?:local )?function (" + Functions.functionNamePattern + ")\\((.*)\\)(?:\\s*::\\s*(.+))?");
 	private static final AtomicBoolean VALIDATE_FUNCTIONS = new AtomicBoolean();
 
 	static {
 		Skript.registerStructure(StructFunction.class,
-			"[:local] function <(" + Functions.functionNamePattern + ")\\((.*)\\)(?:\\s*(?:::| returns )\\s*(.+))?>"
+			"[:local] function <.+>"
```

Comment:
So uh rip `blah() returns blah`, forgot to add `returns` into the new static field

Reply:
nice catch!